src/theme/lexicon.module.scss

```diff
@@ -112,16 +112,20 @@
 			color: $main;
 		}
 
-
 		p {
-			line-height: 1.5;
-			margin-bottom: $base-size * 0.5;
+			line-height: 1.75;
```

Comment:
🤯  thats some line height...

Reply:
I would suggest 1.6 — but dont feel strongly enough that it should change before merging

Reply:
It is ok. It doesn't matter so much. The team is going to review these values at some point soon.